.cargo/nextest.toml

```diff
@@ -0,0 +1,6 @@
+[profile.ci]
+fail-fast = false
+failure-output = "immediate-final"
+
+[profile.ci.junit]
+path = "junit.xml"
```

Comment:
@jclulow it's very easy to tell it to output JUnit, but what should I do with this artifact? There's already a step where we upload stuff someplace, should we do the same with this?

https://github.com/oxidecomputer/omicron/blob/52dfddb9f5c0fe4e73910b387ffd5cd3e1414a89/.github/workflows/rust.yml#L153-L160

Reply:
Yes I think we should add a step for uploading the XML file.  I am not especially familiar with this bit, but it seems like we would want it whether the tests fail or not.

```yaml
    - name: Archive test runner JUnit output
      # actions/upload-artifact@v2.3.1
      uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2
      with:
        name: junit_${{ runner.os }}
        retention-days: 90
        path: |
          junit.xml
```

Something like that maybe?